### ISSUE TITLE
can not get the "embed.sh" example running

### ISSUE THREAD

**Issue by AlexanderCYchu** [facebookresearch/LASER#19]
I follow the instuction README, but still can't get it running.
```
Installation
set the environment variable 'LASER' to the root of the installation, e.g. export LASER="${HOME}/projects/laser"
download encoders from Amazon s3 by bash ./install_models.sh
download third party software by bash ./install_external_tools.sh
```

```
./embed.sh ${LASER}/data/tatoeba/v1/tatoeba.fra-eng.fra fr my_embeddings.raw
 - Encoder: loading /bigdata/facebookresearch-LASER/LASER/models/bilstm.93langs.2018-12-26.pt
 - Tokenizer:  in language fr
 - fast BPE: processing tok
 - Encoder: bpe to my_embeddings.raw
Traceback (most recent call last):
  File "/bigdata/facebookresearch-LASER/LASER/source/embed.py", line 364, in <module>
    buffer_size=args.buffer_size)
  File "/bigdata/facebookresearch-LASER/LASER/source/embed.py", line 295, in EncodeFile
    fin = open(inp_fname, 'r') if len(inp_fname) > 0 else sys.stdin
FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpn0xl23ug/bpe'
```
Thanks for such a great work, There are very little resource in cantonese (yue chinese). This model should be a great help.

**Comment by unexge** [comment]
I had the same problem, this problem is fixed with https://github.com/facebookresearch/LASER/pull/9. you need to pull from master.

**Comment by hoschwenk** [comment]
The problem is fixed with #9 . I'm closing this issue
